crates/modelardb_compression/src/lib.rs

```diff
@@ -14,7 +14,7 @@
  */
 
 //! Compress batches of sorted data points represented by a [`TimestampArray`] and a [`ValueArray`]
-//! using the model types in [`models`](crate::models) to produce compressed segments.
+//! using the model types in [`models`](models) to produce compressed segments.
```

Comment:
Maybe remove "(models)" as it should not be necessary [for rustdoc to create the correct link](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html).

Reply:
Fixed.